### PR TITLE
비밀번호 찾기 페이지 다이얼로그 작업입니다.

### PR DESCRIPTION
**Pull request by chepchep2**
비밀번호 찾기 페이지에서 다이얼로그 작업니다.
구현 영상입니다.
- https://imgur.com/LSGJnwF

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#33** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/33?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#32** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/32?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#31** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/31?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#30** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/30?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#29** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/29?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#28** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/28?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#27** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/27?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#26** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/26?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#25** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/25?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/24?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#23** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/23?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#22** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/22?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#21** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/21?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/20?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#19** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/19?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#18** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/18?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/page/find_password_page.dart [hunk 1]
```diff
@@ -37,6 +38,17 @@ class _FindPasswordPageState extends State<FindPasswordPage> {
     }
   }
 
+  void showResetPasswordDialog(BuildContext context) {
```
Comment (flowkater) [inline lib/page/find_password_page.dart:41]: private 하게 쓸때는 _ 언더바를 prefix 에 항상 붙여주세요.

File: lib/common/auth_dialog.dart [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+import 'package:my_baseball_record/common/app_color.dart';
+import 'package:my_baseball_record/common/app_text_list.dart';
+import 'package:my_baseball_record/common/app_text_style.dart';
+import 'package:my_baseball_record/common/auth_button.dart';
+
+class AuthDialog extends StatelessWidget {
```
Comment (flowkater) [inline lib/common/auth_dialog.dart:7]: AuthDialog 보다는 조금 더 일반화해서 AppDialog 로 만들어주면 좋을 것 같습니다.
다이얼로그 디자인은 여러 곳에서 사용될 수 있을 것 같습니다.

**Review by flowkater — CHANGES_REQUESTED** [review]
다이얼로그 일반화하면 좋을듯 하네요.

File: lib/common/auth_dialog.dart (hunk 2 above)
Comment (chepchep2) [reply lib/common/auth_dialog.dart:7]: 네, 이름 수정하고 더 일반화 시키는 걸 생각해보곘습니다

File: lib/page/find_password_page.dart (hunk 1 above)
Comment (chepchep2) [reply lib/page/find_password_page.dart:41]: 알겠습니다 수정하겠습니다

**Comment by chepchep2** [comment]
### Merge activity

* **Mar 17, 8:12 PM EDT**: @chepchep2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/33).
* **Mar 17, 9:06 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/33) rebased this pull request as part of a merge.
* **Mar 17, 9:07 PM EDT**: @chepchep2 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/33).

**Pull request merged**